### PR TITLE
moving variables from telemetry messages to properties

### PR DESCRIPTION
**Pull request by madeleinefletcher**

### CODE REVIEW

**Comment by madeleinefletcher** [comment]
@MaxShehovtsov

File: JavaScript/JavaScriptSDK/Logging.ts [hunk 1]
```diff
@@ -52,28 +67,51 @@
...
+        public static throwInternalUserActionable(severity: LoggingSeverity, message: _InternalLogMessage) {
             if (this.enableDebugExceptions()) {
                 throw message;
             } else {
-                this.warnToConsole(message);
-                this.logInternalMessage(severity, this.AiUserActionablePrefix + message);
+                if (typeof (message) !== "undefined" && !!message) {
+                    if (typeof (message.message) !== "undefined") {
+                        if (typeof (message.properties) !== "undefined" && !!message.properties) {
+                            this.warnToConsole(message.message + " properties: " + JSON.stringify(message.properties));
```
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/Logging.ts]: Will we still see the "AI" prefix in console?

File: JavaScript/JavaScriptSDK/Logging.ts [hunk 2]
```diff
@@ -52,28 +67,51 @@
...
-        public static throwInternalNonUserActionable(severity: LoggingSeverity, message: string) {
+        public static throwInternalNonUserActionable(severity: LoggingSeverity, message: _InternalLogMessage) {
             if (this.enableDebugExceptions()) {
                 throw message;
             } else {
-                this.warnToConsole(message);
-                this.logInternalMessage(severity, this.AiNonUserActionablePrefix + message);
+                if (typeof (message) !== "undefined" && !!message) {
+                    if (typeof (message.message) != "undefined") {
+                        if (typeof (message.properties) !== "undefined" && !!message.properties) {
```
Comment (MaxShehovtsov) [inline JavaScript/JavaScriptSDK/Logging.ts]: Looks inconsistent with similar check in pollInternalLogs in Initialization.ts, am I missing something?

**Comment by MaxShehovtsov** [comment]
:shipit: with comments

File: JavaScript/JavaScriptSDK/Logging.ts (hunk 1 above)
Comment (madeleinefletcher) [inline JavaScript/JavaScriptSDK/Logging.ts]: Good, point. I'll fix that.

File: JavaScript/JavaScriptSDK/Logging.ts (hunk 2 above)
Comment (madeleinefletcher) [inline JavaScript/JavaScriptSDK/Logging.ts]: Fixed

**Pull request merged**